support-frontend/assets/components/orderSummary/orderSummaryTsAndCs.tsx

```diff
@@ -33,51 +36,62 @@ const productStartDate = css`
 	}
 `;
 
+const guardianWeeklyOrTierThreeProduct = (productKey: ActiveProductKey) => {
+	return [
+		'GuardianWeeklyDomestic',
+		'GuardianWeeklyRestOfWorld',
+		'TierThree',
+	].includes(productKey);
+};
+
 interface OrderSummaryStartDateProps {
 	startDate: string;
+	productKey: ActiveProductKey;
 }
 export function OrderSummaryStartDate({
 	startDate,
+	productKey,
 }: OrderSummaryStartDateProps): JSX.Element | null {
-	return (
-		<ul css={productStartDate}>
-			<li>Your digital benefits will start today.</li>
-			<li>
-				Your Guardian Weekly subscription will start on {startDate}. Please
-				allow 1 to 7 days after your start date for your magazine to arrive,
-				depending on national post services.
-			</li>
-		</ul>
-	);
+	if (guardianWeeklyOrTierThreeProduct(productKey)) {
+		return (
+			<ul css={productStartDate}>
+				{productKey === 'TierThree' && (
+					<li>Your digital benefits will start today.</li>
```

Comment:
I don't think this is right for GW, they don't get digital benefits.

Reply:
I think this does hide the digital benefit for GuardianWeekly only (see pic below)
![image](https://github.com/user-attachments/assets/2f0a0207-7de8-47fd-b6d3-a5b21d0e381c)

Reply:
Oh yes I see now 👍